packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -185,6 +181,14 @@ const useTabs = () => {
       return tab;
     });
 
+    if (!!session.data?.user?.org?.id) {
+      processedTabs.forEach((tab) => {
+        if (tab.href === "/settings/my-account") {
+          tab.children = tab?.children?.filter((child) => child.href !== "/settings/my-account/appearance");
+        }
+      });
+    }
+
```

Comment:
No need to do the iteration again. you can filter the children inside     

const processedTabs = tabs.map((tab) => {
      if (tab.href === "/settings/my-account") {

Reply:
Ahh, sure.